spec/text_message_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'text_message'
+
+describe TextMessage do
+
+  describe "#send_order_text" do
+    it 'can send text message to display order information' do
+      KEYS = {account_sid:'double',auth_token:'double',phone_number:'double'}
+      text = TextMessage.new
+      order = double(:fake_order)
+      allow(order).to receive(:display_order).and_return('order details')
+      module Twilio
+      end
+      class Twilio::REST
+      end
+      class Twilio::REST::Client
+        def initialize(arg1,arg2)
+        end
+        def messages
+          self
+        end
+        def create(from: 'anyone', to: 'anything', body: 'not_empty')
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `from`. You can also write as `create(*)` if you want the method to accept any arguments but don't care about them.<br>Lint/UnusedMethodArgument: Unused method argument - `to`. You can also write as `create(*)` if you want the method to accept any arguments but don't care about them.<br>Lint/UnusedMethodArgument: Unused method argument - `body`. You can also write as `create(*)` if you want the method to accept any arguments but don't care about them.